### PR TITLE
bpf: clean up stale CT-related comments in bpf_lxc

### PR DESCRIPTION
**Pull request by julianwiedmann**
Just some trivial fix-ups, no functional change.

### CODE REVIEW

**Review by borkmann — APPROVED** [review]

File: bpf/bpf_lxc.c [hunk 1]
```diff
@@ -295,7 +294,6 @@ static __always_inline int handle_ipv6_from_lxc(struct __ctx_buff *ctx, __u32 *d
 
 #if defined(ENABLE_L7_LB)
 	if (proxy_port > 0) {
-		/* tuple addresses have been swapped by CT lookup */
```
Comment (pchaigno) [inline bpf/bpf_lxc.c:298]: That comment is still valid. Remember we pass the tuple via the CT_BUFFER map.

File: bpf/bpf_lxc.c [hunk 2]
```diff
@@ -797,7 +795,6 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx, __u32 *d
...
```
Comment (pchaigno) [inline bpf/bpf_lxc.c:800]: Ditto.

**Review by pchaigno — CHANGES_REQUESTED** [review]

File: bpf/bpf_lxc.c (hunk 1 above)
Comment (julianwiedmann) [reply bpf/bpf_lxc.c:298]: yeah - but as we no longer build the tuple ourselves, this sounds confusing now. How about `Address fields in the CT tuple are swapped` ?

Comment (pchaigno) [reply bpf/bpf_lxc.c:298]: > but as we no longer build the tuple ourselves

I'm not sure I'm following. We are still swapping the IP addresses, just in a different BPF program. So the comment still looks correct to me: the CT lookup did swap the tuple addresses.

**Pull request closed without merging**